perfmetrics/scripts/fetch_metrics.py

```diff
@@ -51,15 +88,23 @@ def _parse_arguments(argv):
 
   args = _parse_arguments(argv)
 
+  temp = fio_metrics_obj.get_metrics(args.fio_json_output_path)
+  metrics_data = fio_metrics_obj.get_values_to_upload(temp)
+
   if args.upload_gs:
-    temp = fio_metrics_obj.get_metrics(args.fio_json_output_path, FIO_WORKSHEET_NAME)
-  else:
-    temp = fio_metrics_obj.get_metrics(args.fio_json_output_path)
+    gsheet.write_to_google_sheet(metrics_data, FIO_WORKSHEET_NAME)
```

Comment:
FYI @raj-prince 
Ruchika created one other function named [upload_metrics_to_gsheet](https://github.com/GoogleCloudPlatform/gcsfuse/blob/experiments_dashboard/perfmetrics/scripts/fio/fio_metrics.py#L462), which is nothing but making unnecessary cover on [write_to_google_sheet ](https://github.com/GoogleCloudPlatform/gcsfuse/blob/experiments_dashboard/perfmetrics/scripts/gsheet/gsheet.py#L18). So I didn't copy that function and corresponding unit tests in fio_metrics.py file